packages/mask/src/plugins/Tips/SNSAdaptor/index.tsx

```diff
@@ -0,0 +1,57 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { base } from '../base'
+import { ApplicationEntry } from '@masknet/shared'
+import { useState } from 'react'
+import { TipsEntranceDialog } from './TipsEntranceDialog'
+import { PluginI18NFieldRender } from '@masknet/plugin-infra/content-script'
+import { TipsIcon } from '@masknet/icons'
+
+const sns: Plugin.SNSAdaptor.Definition = {
+    ...base,
+    init(signal) {},
+    ApplicationEntries: [
+        (() => {
+            const name = base.name
+            const icon = <TipsIcon />
+            return {
+                RenderEntryComponent({ disabled, nextIdVerification }) {
```

Comment:
The application entry is totally not business with NextID stuff. Please remove `nextIdVerification`. Alternatively, you can explosure the props of `ApplicationEntry`.

```ts
RenderEntryComponent({ ApplicationEntryProps }) {
    return (
        <>
            <ApplicationEntry
                {...ApplicationEntryProps}
            />
        </>
    )
}
```

Reply:
done